pocs/tests/conftest.py

```diff
@@ -2,39 +2,70 @@
 import pytest
 
 from pocs.utils.config import load_config
-from pocs.utils.data import download_all_files
 from pocs.utils.database import PanMongo
 
 
 def pytest_addoption(parser):
-    parser.addoption(
-        "--hardware-test",
-        action="store_true",
-        default=False,
-        help="Test with hardware attached")
-    parser.addoption(
-        "--camera",
-        action="store_true",
-        default=False,
-        help="If a real camera attached")
-    parser.addoption("--mount", action="store_true", default=False, help="If a real mount attached")
-    parser.addoption(
-        "--weather",
-        action="store_true",
-        default=False,
-        help="If a real weather station attached")
+    parser.addoption("--hardware-test", action="store_true", default=False,
+                     help="Test with hardware attached")
+    parser.addoption("--camera-test", action="store_true", default=False,
+                     help="If a real camera attached")
+    parser.addoption("--mount-test", action="store_true", default=False,
+                     help="If a real mount attached")
+    parser.addoption("--weather-test", action="store_true", default=False,
+                     help="If a real weather station attached")
     parser.addoption("--solve", action="store_true", default=False,
```

Comment:
Why is --solve disabled by default?

And should it be renamed --solve-test or --disable-solver (the latter because it sounds like it is not a test of the solver, but a focus on reducing test duration).

Reply:
It was disabled by default because of the problems getting astrometry.net installed on travis-ci. I think that is resolved now however so we should look at changing this behavior as part of another PR. Created #175